heft-plugins/heft-jest-plugin/src/exports/jest-node-modules-symlink-resolver.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import '../JestRealPathPatch';
+import resolver = require('../HeftJestResolver');
```

Comment:
Use a conventional import?

Reply:
This is cloned from the other resolver, since I assumed it did this for a reason.